### PR TITLE
Test dict change

### PR DESCRIPTION
**Pull request by emord**
follow up for https://github.com/dimagi/Vellum/pull/723

### CODE REVIEW

File: src/xpath.js [hunk 1]
```diff
@@ -4,14 +4,17 @@ define([
     xpath
 ) {
     return {
-        makeXPathModels: function (form) {
+        // hashtagConfig has two properties:
+        //   hashtagDictionary: {hashtag expression: XPath expression}
+        //   hashtagTransformations: {hashtag prefix: function to return property}
+        makeXPathModels: function (hashtagConfig) {
```
Comment (emord) [inline src/xpath.js]: @orangejenny I changed it from `form` to `hashtagConfig`. the initialization of the parser still passes in the form as a form happens to conform to the interface

File: tests/databrowser.js [hunk 2]
```diff
@@ -396,6 +396,29 @@ define([
                         test = JSON.parse($.trim(hashtags.text()));
                     assert.deepEqual(test, {"#case/dob":null});
                 });
+
+                it("the parser and form should point to same hashtag dictionary", function (done) {
```
Comment (emord) [inline tests/databrowser.js]: @millerdev tested.

**Comment by millerdev** [comment]
👍 on ✅

File: src/xpath.js (hunk 1 above)
Comment (orangejenny) [inline src/xpath.js]: sounds reasonable

**Pull request merged**